plugin/mutate/source/dextool/plugin/mutate/backend/database/schema.d

```diff
@@ -1895,6 +2008,43 @@ void upgradeV52(ref Miniorm db) {
     db.run(buildSchema!MutationFileScoreHistoryTable);
 }
 
+// 2022-06-30
+void upgradeV53(ref Miniorm db) {
+    const newFilesTbl = "new_" ~ filesTable;
+    db.run(buildSchema!FilesTbl("new_"));
+    db.run("INSERT INTO " ~ newFilesTbl
+            ~ " (id, path, lang, timeStamp, root, checksum) SELECT id,path,lang,timeStamp,root,checksum0 FROM "
+            ~ filesTable);
+    replaceTbl(db, newFilesTbl, filesTable);
+
+    const newTestFilesTbl = "new_" ~ testFilesTable;
+    db.run(buildSchema!TestFilesTable("new_"));
+    db.run("INSERT INTO " ~ newTestFilesTbl
+            ~ " (id,path,timeStamp,checksum) SELECT id,path,timeStamp,checksum0 FROM "
+            ~ testFilesTable);
+    replaceTbl(db, newTestFilesTbl, testFilesTable);
+
+    const newMutationStatusTbl = "new_" ~ mutationStatusTable;
+    db.run(buildSchema!MutationStatusTbl("new_"));
+    db.run("INSERT INTO " ~ newMutationStatusTbl
+            ~ " (id,status,exitCode,checksum) SELECT id,status,exitCode,checksum0 FROM "
+            ~ mutationStatusTable);
+    replaceTbl(db, newMutationStatusTbl, mutationStatusTable);
+
+    const newDepFileTbl = "new_" ~ depFileTable;
+    db.run(buildSchema!DependencyFileTable("new_"));
+    db.run("INSERT INTO " ~ newDepFileTbl
+            ~ " (id, file, checksum) SELECT id,file,checksum0 FROM " ~ depFileTable);
+    replaceTbl(db, newDepFileTbl, depFileTable);
+
+    const newMarkedMutantTbl = "new_" ~ markedMutantTable;
+    db.run(buildSchema!MarkedMutantTbl("new_"));
+    db.run(
+            "INSERT INTO " ~ newMarkedMutantTbl
```

Comment:
same problem here as previously. All columns must have a value because they are marked as `NOT NULL`.